module/zfs/vdev_queue.c

```diff
@@ -556,6 +547,14 @@ vdev_queue_agg_io_done(zio_t *aio)
 #define	IO_SPAN(fio, lio) ((lio)->io_offset + (lio)->io_size - (fio)->io_offset)
 #define	IO_GAP(fio, lio) (-IO_SPAN(lio, fio))
 
+/*
+ * Sufficiently adjacent io_offset's in ZIOs will be aggregated. We do this
+ * by creating a multilist ABD from the adjacent ZIOs io_abd's. By using
```

Comment:
Let's find another name for "multilist ABD", since this is not related to `multilist_t` / `multilist.c`.  Perhaps "aggregate ABD" or "gang ABD" ("gang block" is something else but "gang" means the same thing here: "Arrange (electrical devices or machines) together to work in coordination." (see [Verb definition 2](https://www.lexico.com/en/definition/gang)))

Reply:
I think gang ABD would be good.